src/plugins/auto/auto_executable_network.cpp

```diff
@@ -57,8 +57,6 @@ IE::Parameter AutoExecutableNetwork::GetMetric(const std::string& name) const {
             return ov::hint::PerformanceMode::LATENCY;
         else if (value == InferenceEngine::PluginConfigParams::CUMULATIVE_THROUGHPUT)
             return ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT;
-        else
-            return ov::hint::PerformanceMode::UNDEFINED;
```

Comment:
For invalid value, why not throw an exception or report error?

Reply:
done